<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_world00">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>503</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world01">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>503</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world02">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>503</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world03">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>503</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>352</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world11">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>352</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world12">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>352</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world13">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>352</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world20">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>201</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world21">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>201</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world22">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>201</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world23">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>201</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world30">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world31">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>50</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world32">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>50</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_world33">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>50</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>160</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_random_ini_world">
      <property name="text">
       <string>随机地图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_game_start">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
